build/Build.md

```diff
@@ -173,19 +173,23 @@ Key takeaways:
 
 <br/>
 
-## 1.6. Vulnerability scanning and container security check
+## <a name="scanning"></a>1.6. Vulnerability scanning and container security check
 
 If a container image includes a vulnerability, then every container generated using the image will also include that vulnerability. Hence it is really important that you scan the newly built image for any vulnerabilities prior to the promotion of the image into a registry.  
 
-Among many other alternative scanning tools, WSO2 uses and would also like to recommend [Clair](https://coreos.com/clair/docs/latest) for this purpose. Clair is an open source project for the static analysis of vulnerabilities in Docker containers. Clair scans each container layer and provides a notification of vulnerabilities that may be a threat, based on the Common Vulnerabilities and Exposures database (CVE) and similar databases from Red Hat, Ubuntu, and Debian.
+Among many other alternative scanning tools, WSO2 uses and would also like to recommend both [Trivy](https://github.com/aquasecurity/trivy) and [Clair](https://coreos.com/clair/docs/latest) for this purpose. 
+
+`Trivy` : is a simple and comprehensive vulnerability scanner for containers and other artifacts. Trivy detects vulnerabilities of OS packages (Alpine, RHEL, CentOS, etc.) and application dependencies (Bundler, Composer, npm, yarn, etc.). 
+
+`Clair` : is an open source project for the static analysis of vulnerabilities in Docker containers. Clair scans each container layer and provides a notification of vulnerabilities that may be a threat, based on the Common Vulnerabilities and Exposures database (CVE) and similar databases from Red Hat, Ubuntu, and Debian.
 
 Additionally, the Center for Internet Security (CIS) has a [Docker Benchmark](https://www.cisecurity.org/benchmark/docker) for evaluating the security of a Docker container.
 
 Docker provides an open source script called [Docker Bench for Security](https://github.com/docker/docker-bench-security). You can use this script to validate a running Docker container against the CIS Docker Benchmark. It can help you verify many items in the CIS Docker Benchmark, but not all items are verifiable with the script. For example, the script cannot verify if the host for the container is hardened or if the container image includes personal data. Review all items in the benchmark and identify those that might need additional verification.
```

Comment:
AFAIK, this tool is validating the Docker environment(Docker daemon) instead of the containers that are running.  Please correct me if I'm wrong.

Reply:
This can do both.